plugin/pkg/admission/webhook/admission.go

```diff
@@ -192,9 +192,12 @@ func (a *GenericAdmissionWebhook) Admit(attr admission.Attributes) error {
 	for i := range hooks {
 		go func(hook *v1alpha1.ExternalAdmissionHook) {
 			defer wg.Done()
+
+			ignoreClientCallFailures := hook.FailurePolicy == nil || *hook.FailurePolicy == v1alpha1.Ignore
+
 			if err := a.callHook(ctx, hook, attr); err == nil {
 				return
-			} else if callErr, ok := err.(*ErrCallingWebhook); ok {
+			} else if callErr, ok := err.(*ErrCallingWebhook); ignoreClientCallFailures && ok {
```

Comment:
Could you print the `ErrCallingWebhook` separately? Currently the printed warning is prefixed with "rejected by webhook ...", which is misleading.

Reply:
> Could you print the ErrCallingWebhook separately? Currently the printed warning is prefixed with "rejected by webhook ...", which is misleading.

updated